### ISSUE TITLE
VNC over SSH disconnects on non-existent timeout

### ISSUE THREAD

**Issue by Vladimir-csp** [FreeRDP/Remmina#859]
Remmina version 1.2.0~rcgit-11+896+next+20160 rebuilt on Debian testing from remmina-next ppa.

VNC over SSH randomly disconnects, debug window shows this:

```
[SSH] tunnel has been disconnected. Reason: ssh_channel_poll() returned an error : Socket error: Connection timed out
[VNC]VNC server closed connection
```

This happens with no apparent hang or waiting, sometimes right in process of visibly active session, so there were no time for any timeout to occur. 
##

**Comment by Kunzol** [comment]
Same ssh_channel_poll error here with remmina 1.2.0-rcgit-18 on Ubuntu xenial while rdp connection.

**Comment by antenore** [comment]
Resolved via #1553 

Let me know otherwise
